### PR TITLE
feat: basic kotlin bindings

### LINKED ISSUE #5: new bindings: kotlin

### PR DESCRIPTION
**Pull request by dignifiedquire**
Closes #5

### CODE REVIEW

File: make_kotlin.sh [hunk 1]
```diff
@@ -0,0 +1,7 @@
+set -eu
+
+# Env
+UDL_NAME="iroh"
+
+# UniFfi bindgen
+cargo run --bin uniffi-bindgen generate "src/$UDL_NAME.udl" --language kotlin  --out-dir ./kotlin
```
Comment (b5) [inline make_kotlin.sh:7]:
```
b5 at number-0 in ~/code/iroh-ffi (feat-kotlin)
$ ./make_kotlin.sh
   Compiling iroh-ffi v0.1.0 (/Users/b5/code/iroh-ffi)
    Finished dev [unoptimized + debuginfo] target(s) in 3.67s
     Running `target/debug/uniffi-bindgen generate src/iroh.udl --language kotlin --out-dir ./kotlin`
Warning: Unable to auto-format iroh.kt using ktlint: Os { code: 2, kind: NotFound, message: "No such file or directory" }
```

Comment (b5) [reply make_kotlin.sh:7]: running `$ curl -sSLO https://github.com/pinterest/ktlint/releases/download/0.50.0/ktlint && chmod a+x ktlint && sudo mv ktlint /usr/local/bin/` from the [kotlin install instructions](https://pinterest.github.io/ktlint/1.0.0/install/cli/) fixed the warning

File: README.md [hunk 2]
```diff
@@ -52,6 +52,8 @@ docker run --rm -v $(pwd):/mnt -w /mnt quay.io/pypa/manylinux2014_x86_64 /mnt/bu
 ## Kotlin
 
 - `make_kotlin.sh` to (re)generate bindings
+- you'll need to install the Android NDK
+- run `rustup target add aarch64-linux-android`
```
Comment (dignifiedquire) [inline README.md:56]: note: this will depend on which android version you target there are many out there..

**Comment by b5** [comment]
stale. closing

**Pull request closed without merging**